jkube-kit/config/image/src/main/java/org/eclipse/jkube/kit/config/image/build/BuildConfiguration.java

```diff
@@ -283,10 +280,9 @@ public boolean optimise() {
         return optimise != null ? optimise : false;
     }
 
-
     @Deprecated
     public String getCommand() {
```

Comment:
Shall we remove the method also?

Reply:
In JKube it's only used in `DockerAssemblyManager`.
I didn't want to remove it because it's part of the public API, but anyway, I don't think anyone is using JKube-kit besides ourselves, so I'll remove.